docs/agents/sematext-agent/processes/metrics.md

```diff
@@ -0,0 +1,65 @@
+title: Process Monitoring
+description: Process Monitoring memory CPU usage
+
+Sematext Agent gathers various metrics and metadata about running processes and brings them to [Sematext Cloud](https://sematext.com/cloud/) so you can painlessly explore the system resources allocated by processes whether they're executing on bare metal machines or scheduled to run on multi-cluster Kubernetes workloads.
+
+Process collector is enabled by default. You can disable the collection of the process's metrics/metadata by setting the following properties:
+
+<div class="mdl-tabs mdl-js-tabs mdl-js-ripple-effect">
+ <div class="mdl-tabs__tab-bar">
+     <a href="#file-enabled" class="mdl-tabs__tab is-active">Config file</a>
+     <a href="#env-enabled" class="mdl-tabs__tab">Env variable</a>
+ </div>
+
+ <div class="mdl-tabs__panel is-active" id="file-enabled">
+   <pre>
+   # st-agent.yml
+   process:
+     enabled: false
+   </pre>
+ </div>
+ <div class="mdl-tabs__panel" id="env-enabled">
+   <pre>
+   PROCESS_ENABLED=false
+   </pre>
+ </div>
+</div>
+
+Similarly, to tweak the frequency of the process's data series delivery to Sematext Cloud adjust the following options:
+
+<div class="mdl-tabs mdl-js-tabs mdl-js-ripple-effect">
+ <div class="mdl-tabs__tab-bar">
+     <a href="#file-interval" class="mdl-tabs__tab is-active">Config file</a>
+     <a href="#env-interval" class="mdl-tabs__tab">Env variable</a>
+ </div>
+
+ <div class="mdl-tabs__panel is-active" id="file-interval">
+   <pre>
+   # st-agent.yml
+   process:
+     interval: 10s
```

Comment:
Too bad we didn't pick a better, more descriptive name for this.  A person not reading the documentation carefully could easily think this is what controls how often process information is _collected_, when in fact this is controlling how often the collected data is _shipped_.

Reply:
This actually controls how often process data is collected, but as soon as collection stage completes, data is shipped as well. So, I would look at this like a single unit of work where data shipping happens right after collection is done.